### ISSUE TITLE
Bookmarklet fails to send

### ISSUE THREAD

**Issue by slightlyannoyed1** [cjlm/send-to-workflowy#12]
Some pages fail to send WF bookmark, seen on github.com and google.com addresses.

`Content Security Policy: The page’s settings blocked the loading of a resource at https://**url-hidden**.netlify.app/send (“connect-src”). javascript:1:232

Uncaught (in promise) TypeError: NetworkError when attempting to fetch resource.`

If this cannot be fixed, it would be good to show a toast notification for success/failure.

**Comment by cjlm** [comment]
Yeah, I think the bookmarklet code could do with an overhaul. I was going to take some inspiration from linkding and/or instapaper. Stretch goal would be a web extension instead.

Can you give me an example of a page for which it errors for you? Is this on Chrome?

**Comment by slightlyannoyed1** [comment]
This is on Firefox, here's the console when I use it on github (for example
this repo)
[image: image.png]
